### PR TITLE
Node.js 10.x Vulnerabilities update to v10.22.1

### PR DESCRIPTION
**Pull request by DalpheSDC**
Node.js 10.x Multiple Vulnerabilities
CVE references
CVE-2020-8252 | CVE-2020-8201 |
Solution
Update to version 10.22.1.

### CODE REVIEW

**Comment by DalpheSDC** [comment]
Hi @damccorm 
Will you look at getting this update in to the coming release?

regards

Peter Dahl

**Comment by DalpheSDC** [comment]
@alex-peck is this change good to be merge to next release?

**Comment by alex-peck** [comment]
@DalpheSDC Can you bump the version to the latest? I believe 10.23.0 is out now.

File: src/Misc/externals.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ L1_MODE=$4
 CONTAINER_URL=https://vstsagenttools.blob.core.windows.net/tools
 NODE_URL=https://nodejs.org/dist
 NODE_VERSION="6.17.1"
-NODE10_VERSION="10.21.0"
+NODE10_VERSION="10.22.1"
+NODE14_VERSION="14.11.0"
```
Comment (alex-peck) [inline src/Misc/externals.sh:11]: Also, we are not shipping Node 14 anymore. Remove this line please :)

Comment (DalpheSDC) [reply src/Misc/externals.sh:11]: @alex-peck 
I have now update the version to 10.23.0

**Review by alex-peck — APPROVED** [review]

File: src/Misc/externals.sh (hunk 1 above)
Comment (alex-peck) [reply src/Misc/externals.sh:11]: Can you delete the NODE14_VERSION entry? Otherwise we are good with this.

**Comment by DalpheSDC** [comment]
Hi @alex-peck 
I have remove the nodejs 14 entry.
regards
Peter Dahl

**Pull request merged**